content/en/functions/strings.ToUpper.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
The already documented "upper" function is an alias for strings.ToUpper. You can remove this page.

You are welcome to add a second signature to the existing function.

Reply:
Page removed and second signature added to the existing function.